tests/layer_tests/pytorch_tests/test_matmul.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import pytest
+import torch
+from pytorch_layer_test_class import PytorchLayerTest
+
+class TestMVOperation(PytorchLayerTest):
+    def _prepare_input(self, matrix, vector):
+        matrix_input = np.array(matrix).astype(np.float32)
+        vector_input = np.array(vector).astype(np.float32)
+        return [matrix_input, vector_input]
+
+    def create_model(self, matrix, vector):
+        class CustomMVOperation(torch.nn.Module):
+            def forward(self, matrix, vector):
+                return torch.mv(matrix, vector)  # Using torch.mv for matrix-vector multiplication
```

Comment:
Types here are not known. Tests will fail because of that. Please use `to` torch function to convert inputs to required dtype. Also please add more test with mixed types if torch.mv supports it